src/background/tab-manager.ts

```diff
@@ -127,7 +127,6 @@ export default class TabManager {
                         frameURL: frameId === 0 ? null : senderURL,
                     });
                     this.stateManager.saveState();
-                    sendResponse({type: '¯\\_(ツ)_/¯'});
```

Comment:
`reply()` is called on line 125 which calls `chrome.tabs.sendMessage` on line 90 prior to this call to `sendResponse()`. I fixed this bug by calling `sendResponse()` in `makeChromiumHappy()`, but we could do the same by moving this `sendResponse()` to before `reply()`. It really does not matter much.